### PR TITLE
Begin moving pkgs w/ deps on pkg/oc

### PR DESCRIPTION
**Pull request by juanvallejo**
This patch solves a few of the items (currently checked) from https://github.com/openshift/origin/pull/17309

cc @deads2k @openshift/cli-review @liggitt

### CODE REVIEW

File: pkg/build/generator/generator.go [hunk 1]
```diff
@@ -25,7 +25,6 @@ import (
 	"github.com/openshift/origin/pkg/cmd/server/bootstrappolicy"
 	imageapi "github.com/openshift/origin/pkg/image/apis/image"
 	imageclient "github.com/openshift/origin/pkg/image/generated/internalclientset/typed/image/internalversion"
-	"github.com/openshift/origin/pkg/oc/admin/policy"
```
Comment (deads2k) [inline pkg/build/generator/generator.go]: Who depends on pkg/build/generator?  Anyone outside of oc?

**Comment by deads2k** [comment]
This looks pretty good to me.  Check for conflicts with the rebase branch.

**Comment by juanvallejo** [comment]
@deads2k 
> Who depends on pkg/build/generator? Anyone outside of oc?

Found the following files, outside of `pkg/oc`, with references to `pkg/build/generator`:

- pkg/image/controller/trigger/image_trigger_controller_test.go
- pkg/build/apiserver/apiserver.go
- pkg/build/controller/buildconfig/buildconfig_controller.go
- pkg/build/controller/build/build_controller.go
- pkg/build/registry/buildconfiginstantiate/rest_test.go
- pkg/build/registry/buildconfiginstantiate/rest.go
- pkg/build/registry/buildclone/rest_test.go
- pkg/build/registry/buildclone/rest.go

**Comment by deads2k** [comment]
ok, then that helper duplication is fine.

**Comment by juanvallejo** [comment]
@deads2k 
> This looks pretty good to me. Check for conflicts with the rebase branch.

Thanks, this caused no conflict locally with the rebase branch

**Comment by deads2k** [comment]
> Thanks, this caused no conflict locally with the rebase branch

Ok.  Get green, then re-check.  If you're still clear I'll merge.

**Comment by juanvallejo** [comment]
flaked on https://github.com/openshift/origin/issues/17394

**Comment by deads2k** [comment]
needs a squash, then lgtm

**Comment by juanvallejo** [comment]
@deads2k thanks, commit squashed

**Pull request merged**